csaf_2.0/prose/csaf-v2-editor-draft.md

```diff
@@ -1655,7 +1655,7 @@ If an issuing party decides to change its Namespace it SHOULD reissue all CSAF d
 Example:
 
 ```
-    http://www.example.com
+    https://www.example.com
```

Comment:
Is it required to have `https` for the `namespace`? If so, should we state that explicitly? Or do we guide by example?

Reply:
The latter is my intention. These non-authentic URLs are mostly accidental or historical ballast. I suggest, we do strongly emphasize authenticity and cryptographic hardness, so, we should maintain that level for many references that are extensions of the CSAF document (by reference).

Reply:
Agreed. Thanks for clarifying that.